storm/tests/compose/Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM maven:3.5.0-jdk-7
 
-RUN curl -fsSL https://github.com/platinummonkey/static_ci_files/raw/master/datadog/integrations-extras/apache-storm-1.2.1.tar.gz -o storm.tar.gz \
```

Comment:
Question.

Any reason not using the release archives (https://github.com/apache/storm/releases) ?  e.g. `https://github.com/apache/storm/archive/v1.2.3.tar.gz`

Probably lighter to download than the git repository (and maybe more stable too).

Reply:
the mirror links are not reliable, they've been dropped before $0.02